sink-common/src/configuration.rs

```diff
@@ -49,6 +49,9 @@ pub struct ConfigurationArgs {
     /// Jsonnet transformation to apply to data. If it starts with `@`, it is interpreted as a path to a file.
     #[arg(long, env)]
     pub transform: Option<String>,
+    // Limits the maximum size of a decoded message.
```

Comment:
Needs `///` to make it a doc comment.

```suggestion
    /// Limits the maximum size of a decoded message. Accepts message size in human readable form,
    /// e.g. 1KB, 1MB, 1GB. If not set, the default is 1MB.
```